__main__.py

```diff
@@ -44,15 +44,18 @@
 directory = os.getcwd()
 filename = directory + "/"
 if args.filename:
-    filename += args.filename
+    if args.filename[0] == "/":
+      filename = args.filename
+    else:
+      filename += args.filename
 
 if args.displaywidth:
     displaywidth = args.displaywidth
 else:
     displaywidth = None
 
 # running jobs
```

Comment:
maybe convert args.jobtype here to `.upper()` so you dont need it one every line